docs/source/fundamentals/interfaces.rst

```diff
@@ -37,6 +37,18 @@ executed at the top of your session. Please note:
 
 - This loads the extension to the current session only, i.e., it does not carry over to different sessions; you will need to repeat it for each new session.
 
+.. note::
```

Comment:
@mingjerli I moved Jupyter Hub instructions to a more relevant section. Please let me know if you have questions about this change.